packages/node_modules/@ciscospark/react-container-notifications/src/selectors.js

```diff
@@ -2,34 +2,15 @@ import {createSelector} from 'reselect';
 
 import {NOTIFICATION_TYPE_OTHER, NOTIFICATION_TYPE_POST} from './actions';
 
-const getActivities = (state) => state.activities.byId;
 const getNotifications = (state) => state.notifications.get('items');
 const getNotificationSettings = (state) => state.notifications.get('settings');
-const getAvatars = (state) => state.avatar.get('items').toJS();
-
-function findActivityWithUrl(activities, url) {
-  if (activities) {
-    return activities.find((activity) => activity.url === url);
-  }
-  return {};
-}
 
 const getUnsentNotifications = createSelector(
-  [getActivities, getNotifications, getAvatars],
-  (activities, notifications, avatars) =>
+  [getNotifications],
+  (notifications) =>
     notifications
       .map((notification) => {
-        if (notification.type === NOTIFICATION_TYPE_POST) {
-          const notificationActivity = findActivityWithUrl(activities, notification.notificationId);
-          if (notificationActivity) {
-            return Object.assign({}, notification, {
-              username: notificationActivity.actor.displayName,
-              message: notificationActivity.object.displayName,
-              avatar: avatars[notificationActivity.actor.id]
-            });
-          }
-        }
-        if (notification.type === NOTIFICATION_TYPE_OTHER) {
+        if (notification.type === NOTIFICATION_TYPE_OTHER || notification.type === NOTIFICATION_TYPE_POST) {
```

Comment:
Since these are the only two notification types, let's remove the type system altogether.

Reply:
Oh gotcha, wasn't sure if they were the only two or not. Thanks!